entities/weapons/pocket/cl_menu.lua

```diff
@@ -32,14 +32,27 @@ function DarkRP.openPocketMenu()
     if not wep:IsValid() or wep:GetClass() ~= "pocket" then return end
     if not pocket then pocket = {} return end
     if table.Count(pocket) == 0 then return end
-    frame = vgui.Create("DFrame")
-
+    
+	local count = GAMEMODE.Config.pocketitems or GM.Config.pocketitems
```

Comment:
You're using tabs for indentation, even though the rest of the file is using spaces